documentatie/flowcharts/date-entry-navigation.md

```diff
@@ -0,0 +1,149 @@
+# Data Entry Navigation
+
+## Questions
+- Where does "cache input" happen in these flows?
+- Is there a different warning icon for accepted versus not-accepted warnings? Or is it not possible to leave a page without accepting the warning? Also, accepting warnings is not yet included in any of the flows.
+
+
+## Render page
+
+Render happens based on last received API response.
+
+```mermaid
+flowchart TD
+
+    %% elements
+    flow-start([start])
+    flow-end([end])
+    illegal-state{{illegal state}}
+
+    show-error
+    error-icon-next-page
+    %% error-icon-prev-page is illegal state
+    error-icon-cur-page
+
+    show-warning
+    warning-icon-next-page
+    warning-icon-prev-page
+    warning-icon-cur-page
+
+    error-next-page{error-next-page?}
+    error-prev-page{error-prev-page?}
+    error-cur-page{error-cur-page?}
+
+    warning-next-page{warning-next-page?}
+    warning-prev-page{warning-prev-page?}
+    warning-cur-page{warning-cur-page?}
+
+    cur-page-submitted{cur-page-submitted?}
+
+    fill-cached-input
+
+    %% flow
+    flow-start --> error-prev-page
+
+    error-prev-page -- yes --> illegal-state
+    error-prev-page -- no --> error-next-page
+
+    error-next-page -- yes --> error-icon-next-page
+    error-icon-next-page --> error-cur-page
+    error-next-page -- no --> error-cur-page
+
+    error-cur-page -- yes --> cur-page-submitted
+    cur-page-submitted -- yes --> error-icon-cur-page
+    error-icon-cur-page --> show-error
```

Comment:
the current page never has an icon other then --> in the nav menu. No errors, no warnings. Just the -->

Reply:
fixed